src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -42,12 +42,12 @@ public void testAddUsers() {
         ctl.addUser(
             ds, /*userID=*/ 1l, /*AuthID=*/ 2l,
             /*userName=*/ "Sarah", /*userYear=*/ 2022,
-            /*userMajors=*/ user1Majors, /*skills=*/ Skills.LEADERSHIP,
+            /*userMajors=*/ user1Majors, /*skills=*/ Skills.LEADERSHIP.name(),
```

Comment:
[ERROR] /home/talsternberg/project-management-tool/src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java:[45,76] incompatible types: java.lang
.String cannot be converted to com.rtb.projectmanagementtool.user.Skills

Reply:
I think you can remove the `.name()`.

Reply:
When I eliminate .name(), I get a test failure saying: "Tests in error:
  testAddUsers(com.rtb.projectmanagementtool.user.UserControllerTest): skills: com.rtb.projectmanagementtool.user.Skills is not a supported property type."

Reply:
This is probably from line 61 on your `toEntity` method.  skills at that point is still the enum, which Datastore does not support.  You'll need to convert to string when building your Entity, so Datastore can store it as a string.